The linux-awareness layer doesn't auto-activate when the kernel has been built with
O=<objdir>. Removing this redundant lookup solves the problem.

Signed-off-by: Chris Smith <chris.smith@st.com>
Index: gdb-6.5/gdb/linux-awareness.c
===================================================================
--- gdb-6.5.orig/gdb/linux-awareness.c	2007-10-08 15:42:04.000000000 +0100
+++ gdb-6.5/gdb/linux-awareness.c	2007-10-08 15:42:05.000000000 +0100
@@ -6123,7 +6123,6 @@
 	&& linux_awareness_auto_activate_lookup_symtab ("block_dev.c")
 	&& linux_awareness_auto_activate_lookup_symtab ("vmalloc.c")
 	&& linux_awareness_auto_activate_lookup_symtab ("page_alloc.c")
-	&& linux_awareness_auto_activate_lookup_symtab ("fs/buffer.c")
 	&& linux_awareness_ops->lo_check ())
 	autodetection = LINUX_WITH_DEBUGINFO;
 
